src/components/ConfirmPopover.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
```

Comment:
I think we already have a ConfirmModal. Can't we reuse that?

Reply:
My thinking is that it makes sense to have both a popover and modal as they serve different purposes. 

How would you recommend we re-use the ConfirmModal in this case?

Reply:
Most of the code is the same apart from:
1. Modal => Popover
2. AnchorPositions.

Maybe we can create a small component for the ConfirmModal's content. Then, add `anchorPosition` prop. 
Whenever the user has passed anchorPosition switch between containers. 
From
```
   <Modal
        onSubmit={props.onConfirm}
        onClose={props.onCancel}
        isVisible={props.isVisible}
        type={props.isSmallScreenWidth
            ? CONST.MODAL.MODAL_TYPE.BOTTOM_DOCKED
            : CONST.MODAL.MODAL_TYPE.CONFIRM}
    >
```
to 
```
    <Popover
        onSubmit={props.onConfirm}
        onClose={props.onCancel}
        isVisible={props.isVisible}
        anchorPosition={props.anchorPosition}
    >
    ```

Reply:
We can also create two components for Container JSX.

Reply:
Gotcha. While that does save us some code, I think that having 2 different components for different purposes is fine in this case. 

I tried what you suggested by moving the buttons into their own component, and then using that component in each of these files and realized that the styles are very very different for the buttons and IMO it is not worth trying to consolidate the code.